Networking/NetworkingTests/Mapper/OrderShippingLabelListMapperTests.swift

```diff
@@ -88,4 +88,16 @@ final class OrderShippingLabelListMapperTests: XCTestCase {
         XCTAssertEqual(response.settings, .init(siteID: sampleSiteID, orderID: sampleOrderID, paperSize: .label))
         XCTAssertEqual(response.shippingLabels.count, 1)
     }
+
+    func test_order_shipping_labels_and_settings_are_properly_parsed_if_labelsData_contains_an_error() throws {
```

Comment:
nit: I think we should rename this test to make it explicit that we are testing that labels with `error` are filtered while mapping.

Reply:
I renamed it to `test_order_shipping_labels_mapper_filters_out_labels_with_error_in_labelsData`, it should be clearer now f5246114786b9860b3ae72faf26871ab011a20b3